specs/phase0/p2p-interface.md

```diff
@@ -547,7 +547,8 @@ Requests count beacon blocks from the peer starting from `start_slot`, leading u
 The request MUST be encoded as an SSZ-container.
 
 The response MUST consist of zero or more `response_chunk`. Each _successful_ `response_chunk` MUST contain a single `SignedBeaconBlock` payload.
-Clients MUST support requesting blocks since the start of the weak subjectivity period and up to the given `head_block_root`.
+
+Clients MUST keep a record of signed blocks seen since the since the start of the weak subjectivity period and MUST support requesting blocks up to the given `head_block_root`.
```

Comment:
```suggestion
Clients MUST keep a record of signed blocks seen since the since the start of the weak subjectivity period and MUST support serving requests of blocks up to their own `head_block_root`.
```

Wording was strange here, and I think we should be defining what the responder must support? @djrtwo

Reply:
looks good